common/unsafe/src/test/java/org/apache/spark/unsafe/PlatformUtilSuite.java

```diff
@@ -134,4 +135,24 @@ public void memoryDebugFillEnabledInTest() {
       MemoryAllocator.MEMORY_DEBUG_FILL_CLEAN_VALUE);
     MemoryAllocator.UNSAFE.free(offheap);
   }
+
+  @Test
+  public void heapMemoryReuse() {
+    MemoryAllocator heapMem = new HeapMemoryAllocator();
+    // The size is less than 1M,allocate new memory every time.
```

Comment:
Is it better to refer to `HeapMemoryAllocator.POOLING_THRESHOLD_BYTES` at `1M`?

Reply:
Yeah,i agree with you,thanks